recipes/fast-cdr/all/conanfile.py

```diff
@@ -131,6 +120,8 @@ def package_info(self):
         self.cpp_info.names["cmake_find_package"] = "fastcdr"
         self.cpp_info.names["cmake_find_package_multi"] = "fastcdr"
         self.cpp_info.libs = tools.collect_libs(self)
+        if self.settings.os == "Windows" and self.options.shared:
+            self.cpp_info.defines.append("FASTCDR_DYN_LINK")
         self._create_cmake_module_alias_target()
```

Comment:
```suggestion
        self._create_cmake_module_alias_targets()
```

Reply:
This needs be in package

Reply:
ok got it - everything that creates files etc. comes to package. everything that is just meta information to artifacts - to info